### PR TITLE
make faster

### PR DESCRIPTION
**Pull request by ProkopRandacek**

### CODE REVIEW

**Comment by mvolfik** [comment]
The commit message should contain a better explanation

**Review by mvolfik — CHANGES_REQUESTED** [review]
I suggest explaining more in the commit message

Force-pushing to this branch should be ok

File: test2.c [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#include <stdio.h>
+
+int main(void) {
+    for (int i = 0; i < 15; i++) {
```
Comment (marekmaskarinec) [inline test2.c:4]: You should remove `{` and `}`.

**Comment by ProkopRandacek** [comment]
I believe that the commit message explains the suggested changes in appropriate detail for the changes made.
gif

**Comment by xiaoxiae** [comment]
.

**Pull request merged**